### DISCUSSION TITLE
Q/FR: Arduino RP2400 Atecc support

### DISCUSSION THREAD

**Discussion by mattmeye** [micropython/micropython#15809] [Libraries & Drivers]
Hello, First let me thank you for your amazing project. 

I have a question, is a support for Atecc chip in the Arduino rp2400 planned?

I would like to create and save certs for iot communication safely.

Kind regards, matt

**Comment by dpgeorge** [comment]
@iabdalkader do you have any plans to implement a driver for the ATECC608A?

**Comment by iabdalkader** [comment]
There's already a library for micropython that looks very promising: https://github.com/dmazzella/ucryptoauthlib. As you can see it's a lot of work, so it doesn't make sense to rewrite it. If it's add as a submodule, somewhere, we can just include it in the board manifest.

**Comment by mattmeye** [comment]
Sounds good, I would appreciate that!

**Comment by mattmeye** [comment]
I have no experience with Micropython, but could try to support with a few hints. Is there already a library that has been integrated via submodule?

**Comment by iabdalkader** [comment]
> Is there already a library that has been integrated via submodule?

No.
